frontends/cli/source/cli_frontend.d

```diff
@@ -109,24 +109,29 @@ DeveloperSession login(Device device, ADI adi, bool interactive) {
     // ...
 
     if (account) return null;
-    if (!interactive) {
+    string appleId = environment.get("APPLE_ID_USER");
+    string password = environment.get("APPLE_ID_PWD");
+    if(interactive && (appleId is null || password is null)) {
```

Comment:
I would probably do an `interactive || ` here, as if the interactive switch is given it's probably to make the prompt appear.

Reply:
I think `&&` is the better choice, as we want the prompt only in interactive mode - if there is no interactive mode, don't prompt, but fail - otherwise you'd receive a prompt in an non-interactive session. 

For my use case I want to use this in an autonomous scripting scenario - failing in case input is required is exactly what I want :D

Reply:
Maybe just interactive then?